packages/editor/src/components/block-edit/edit.native.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import { omit } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { withFilters } from '@wordpress/components';
+import { getBlockType } from '@wordpress/blocks';
+
+export const Edit = ( props ) => {
+	const { attributes = {}, name } = props;
+	const blockType = getBlockType( name );
+
+	if ( ! blockType ) {
+		return null;
+	}
+
+	const style = props.style || attributes.style;
+	const Component = blockType.edit;
+
+	return (
+		<Component
+			{ ...omit( props, [ 'className' ] ) }
+			style={ style }
+		/>
+	);
+};
+
+export default Edit;
```

Comment:
I think you will need:
`export default withFilters( 'editor.BlockEdit' )( Edit );`

to make sure that hooks work properly.

Reply:
Isn't `withFilter` used to to customize the component, I think my goal here was to avoid side effects but I don't mind calling it since it probably doesn't have any filters at the moment.

Reply:
It's a way to apply Higher-Order components, so in that sense, it allows to customize the component. At some point you will need it to have proper support for block alignments, handling custom class names and anchors.

Reply:
👍 Do you think it's necessary at this point for mobile?
For instance I didn't import those as well as I don't think there necessary at this point:
```
import '@wordpress/notices';
import '@wordpress/nux';
import '@wordpress/rich-text';
import '@wordpress/viewport';
```

Reply:
You can try to skip it. If it works you can add it back later.